CMakeLists.txt

```diff
@@ -198,6 +199,14 @@ if(WIN32)
     add_definitions(-DOIS_DYNAMIC_LIB)
 endif()
 
+if(OD_USE_SFML_WINDOW)
+    if(WIN32)
+        message(WARNING "SFML window is not implemented on windows yet!")
```

Comment:
It's just in our project, there are some small differences when the window is created, and I haven't done the windows specific part yet. (see http://www.ogre3d.org/forums/viewtopic.php?f=1&t=79384#p499268)